js_basics/task5.js

```diff
@@ -0,0 +1,49 @@
+const log = console.log
+import chalk from 'chalk';
+
+//5.1
+const subtask1 = 5.1
+
+//задаємо параметри кола
+const radius = 7
+
+//обчислюємо значення
+const PloshiaKola = Math.PI*Math.pow(radius,2)
+
+//округлюємо значення
+const PloshiaKolaRound = Math.round(PloshiaKola)
```

Comment:
use` .toFixed()`

The `Math.round()` static method returns the value of a number rounded to the nearest integer.